armi/utils/mathematics.py

```diff
@@ -290,6 +291,39 @@ def getStepsFromValues(values, prevValue=0.0):
     return steps
 
 
+def isMonotonic(inputList, relation):
```

Comment:
Minor things:

1. I believe `inputList` is a bit mis-named, since this method works for any iterable. I might do `inputIter` or something.
2. I am not super thrilled with the long names for the inputs: "increaseOrEqual", etc.  Why not just ">=", as that is a lot faster to write, and more standard in Python. Or, Hell, just let people pass in the actual function `>`?
3. Very Minor: But I think I would pull out your dictionary mapping strings to operators, before the `try` loop. You are mixing data (your dictionary) with logic (the `try`/`accept`).

Sorry!

Reply:
Done.